slurm-convert-tifs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#SBATCH --account=su62
+#SBATCH --time=00:40:00
+#SBATCH --ntasks=1
+#SBATCH --mem-per-cpu=512
+#SBATCH --cpus-per-task=1
+# SBATCH --mail-user=juan.nunez-iglesias@monash.edu
+# SBATCH --mail-type=FAIL
+echo $DIR
+/home/jnun0003/.conda/envs/mic/bin/python ./convert-tifs.py $DIR
```

Comment:
It'd be better if you could put these scripts in the shared project folder, rather than your own space. Feel free to lock down permissions in the shared space if you like, but this way there'll be less tedious editing before people can run it.